PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -8561,6 +8841,27 @@ void Trace(const char* functionName, Int_t lineNumber)
 
 //============================================================
 
+void StartFunction(const char* functionName)
+{
+  // A simple utility wrapper, used when tc.fVerbose = kTRUE. It merely ensures uniform formatting of notification when the function starts.
+
+  LOGF(info, "\033[1;32mStart %s\033[0m", functionName); // prints in green
+
+} // void StartFunction(const char* functionName)
+
+//============================================================
+
+void ExitFunction(const char* functionName)
+{
+  // A simple utility wrapper, used when tc.fVerbose = kTRUE. It merely ensures uniform formatting of notification when the function exits.
+
+  LOGF(info, "\033[1;32mExit %s\033[0m\n", functionName); // prints in green
+  sleep(0.1);
```

Comment:
Is this really needed in here?
I don't know if we are allowed to use `sleep`
Especially after a `LOGF` which is automatically handled by the Fair log framework
I need to check it

Reply:
This was used only during debugging, but I have removed it now.